lib/fs/promises.js

```diff
@@ -145,17 +145,20 @@ async function readFileHandle(filehandle, options) {
   if (size > kMaxLength)
     throw new ERR_FS_FILE_TOO_LARGE(size);
 
-  const chunks = [];
-  const chunkSize = Math.min(size, 16384);
-  const buf = Buffer.alloc(chunkSize);
-  let totalRead = 0;
-  do {
-    const { bytesRead, buffer } =
-      await read(filehandle, buf, 0, buf.length);
-    totalRead = bytesRead;
-    if (totalRead > 0)
-      chunks.push(buffer.slice(0, totalRead));
-  } while (totalRead === chunkSize);
+    const chunks = [];
```

Comment:
This indentation looks off.

Reply:
You're right, I forgot to re-run the tests after TimothyGu change request. My bad!